apps/www/src/registry/default/plate-ui/link-floating-toolbar.tsx

```diff
@@ -71,26 +71,31 @@ export function LinkFloatingToolbar({ state }: LinkFloatingToolbarProps) {
   if (hidden) return null;
 
   const input = (
-    <div className="flex w-[330px] flex-col">
+    <div
+      className="flex w-[330px] flex-col"
+      onKeyDownCapture={(e) => {
+        if (e.key === "Enter" || e.keyCode === 13) {
+          e.preventDefault();
+        }
+      }}
```

Comment:
@zbeyens Sure I'll test it out today to make sure it works the same :)

Reply:
@zbeyens if we add it to the `FloatingLinkUrlInput` it will not capture the event that the user presses enter after entering text into the label input field and will invoke the errant behaviour of calling the parent form's submit method. As such I think the most pragmatic approach is either leave it where it is one of: 

- Move it to the next child down the tree (as this wraps both input fields)
- Add it to both input fields (seems a bit excessive and harder to manage)

Reply:
Makes sense. Since we try hard to follow shadcn pattern, I think adding a new hook `useFormInputProps` in `@udecode/plate-utils` would be optimal, that should be used by any editor input (your second suggestion).

Reply:
@zbeyens no probs at all, I will try get something pulled together over this weekend :)